lib/stages/rawFileWrite.hpp

```diff
@@ -37,6 +37,7 @@ class rawFileWrite : public kotekan::Stage {
     std::string base_dir;
     std::string file_name;
     std::string file_ext;
+    uint32_t _num_frames_per_file;
```

Comment:
This needs to be documented (here and in the list of options for this stage).   Also if we are going to add this functionality, we should also add the ability to read these files back with rawFileRead.  Ideally `rawFileRead` shouldn't need to know how many frames there with another `_num_frames_per_file` but be able to figure that out based on how long the file is.

Reply:
Where do you mean for the list of options?